platform/metrics/platform-metrics-reporting/src/main/java/ddf/metrics/reporting/internal/rrd4j/RrdMetricsRetriever.java

```diff
@@ -325,7 +325,7 @@ public String createCsvData(String rrdFilename, long startTime, long endTime)
 
     MetricData metricData = getMetricData(rrdFilename, startTime, endTime);
 
-    StringBuffer csv = new StringBuffer("");
+    StringBuilder csv = new StringBuilder("");
```

Comment:
✏️  Probably should just be using the default ctor here or if we know the normal size will be bigger than 16 than using a ctor with a closer approximate size would be better.